libs/@local/hashql/ast/src/lib.rs

```diff
@@ -52,7 +52,8 @@
     never_type,
     decl_macro,
     macro_metavar_expr,
-    let_chains
+    let_chains,
```

Comment:
I wonder why clips does not complain here, isn't that stabilized already?

Reply:
I am guessing that clippy is maybe just a week behind, we currently have 4 places that use said feature, maybe it'd make sense to create a follow-up PR or just wait for next week to make the change with the next toolchain bump.